indica/equilibrium.py

```diff
@@ -161,6 +162,8 @@ def __init__(
             np.arctan2(self.zmin - self.zmag, self.Rmin - self.rmag) % (2 * np.pi),
         ]
 
+        # Mock of sess.prov needed as a fix for a recurring error
+        # (probably also a future issue)
```

Comment:
Is this still the case. If so, is it only a problem you've had when running unit tests or have you noticed it in more realistic tests of the code on real data too?

Reply:
Sorry, this is a leftover comment from before I put in the fix you suggested (`sess=MagicMock()` keyword argument).

Reply:
Happens to everyone.